modules/publication/jsx/viewProject.js

```diff
@@ -87,60 +79,66 @@ class ViewProject extends React.Component {
    */
   fetchData() {
     let self = this;
```

Comment:
Hi @laemtl, I believe `let self = this` isn't necessary with your changes.

Reply:
Your totally right, thanks for catching this!

Reply:
@maltheism Fixed.